database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -15,6 +15,7 @@ public function up(): void
             $table->id();
             $table->string('name');
             $table->string('email')->unique();
+            $table->integer('role');//Admin->1,Subscriber->0
```

Comment:
since you are using integer to depict roles. 

make this default 1, so by default all users are subscribers. Until they are made admins